metadata-db-app/src/cmr/metadata_db/services/generic_documents.clj

```diff
@@ -32,7 +32,7 @@
    know what they are doing."
   [context params provider-id document]
   (let [db (mdb-util/context->db context)
-        document-as-map (json/parse-string document true)
+        document-as-map document
```

Comment:
I really thought I needed to parse the document, that it was a string and not a map. Will this change on merge?